README.md

```diff
@@ -245,44 +219,27 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Cindy Shin for the design template provided.
+Originally designed by: [Cindy shin]{https://www.behance.net/adagio07}
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] You did well at writing about the original design,but I see that the link for the design is broken,so please try to use the correct link as stated by the previous reviewer 😊